test/PostgreSQL/Test.hs

```diff
@@ -1227,6 +1232,197 @@ testCommonTableExpressions = do
             pure res
         asserting $ vals `shouldBe` fmap Value [2..11]
 
+testPostgresqlLocking :: SpecDb
+testPostgresqlLocking = do
+    describe "Monoid instance" $ do
+      let toText conn q =
```

Comment:
please use 4 space indent thoughout this project

Reply:
Thanks, fixed!